plugins/modules/aci_cloud_ctx_profile.py

```diff
@@ -50,10 +50,11 @@
     description:
     - The name of the cloud region in which to deploy the network construct.
     type: str
-  vpn_gateway:
+  cloud:
     description:
-    - Determine if a VPN Gateway Router will be deployed or not.
-    type: bool
+    - The vendor of the controller
+    choices: [ aws, azure ]
+    type: str
```

Comment:
shouldn't it be required?

Reply:
I suppose it would only be required in 'present' status so that I don't add required here

Reply:
I don't think it is required at all right now. Should be required for present and absent at least.

Reply:
actually in our code, cloud is only used in present state  when we add child 'cloudRsCtxProfileToRegion' to ctx_profile object. (line 285)
as for absent state, we don't use a cloud vendor, we just use tenant and ctx_profile name to specify the object we want to delete.